src/controller_examples/rabbitmq_controller/exec/reconciler.rs

```diff
@@ -38,12 +39,12 @@ impl RabbitmqReconcileState {
 pub struct RabbitmqReconciler {}
 
 #[verifier(external)]
-impl Reconciler<RabbitmqCluster, RabbitmqReconcileState, EmptyMsg, EmptyMsg, EmptyLib> for RabbitmqReconciler {
+impl Reconciler<RabbitmqCluster, RabbitmqReconcileState, EmptyType, EmptyType, EmptyLib> for RabbitmqReconciler {
```

Comment:
Nice. I was about to ask you to rename these two. Actually, can you also rename `EmptyLib`?

Reply:
How about `EmptyAPI`?

Reply:
Yes